drivers/CoreDACTouch/include/external/MCP4728.h

```diff
@@ -0,0 +1,100 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "cstdint"
+
+namespace leka::mcp4728 {
+
+namespace command {
+
+	inline constexpr uint8_t writeAllInputRegister			  = 0x00;
```

Comment:
même chose que dans #792 pour ici et pour tous les autres:

```suggestion
	inline constexpr auto writeAllInputRegister			  = uint8_t {0x00};
```